examples/demo-windows.yaml

```diff
@@ -0,0 +1,34 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: podpvc-windows
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: csi-gce-pd
+  resources:
+    requests:
+      storage: 1Gi
+
+---
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: web-windows
+spec:
+  tolerations:
```

Comment:
Is a blanket toleration a good idea? What about just tolerating the Windows taint? Was there not an admission webhook that automatically adds the toleration?

Reply:
At any rate documenting the assumption about what windows taints are expected might be a good idea?

Reply:
In OSS, acccording to code https://github.com/kubernetes/kubernetes/search?q=WINDOWS_NODE_TAINTS, test cluster set "node.kubernetes.io/os=win1809:NoSchedule". I think this should be updated for different os windows cluster. follow up with an issue to track it.

In GKE, the related doc is here https://cloud.google.com/kubernetes-engine/docs/how-to/deploying-windows-app
It does not contain any os version information.

Reply:
From the doc, it looks like a toleration is automatically applied by the admission controller? Do we need the pod to explicitly set it?

Reply:
I think it will be added in GKE, but not gce cluster.